### PR TITLE
run-one 1.18 (new formula)

### PR DESCRIPTION
**Pull request by fgregg**
Initial formula for the run-one bash scripts to help you run one
and only command with args. (standard part of the ubuntu distro)

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/run-one.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class RunOne < Formula
+  desc "Run one instance of a unique combination of command and args at a time"
+  homepage "https://launchpad.net/run-one"
+  url "https://github.com/datamade/run-one/archive/ea3d7744043bf737aa52c3093f068a49b7621d02.tar.gz"
+  version "1.18"
+  sha256 "8094b26db3efca2a419737ab26d1926e98c116d9e687dc32cd841192c3192dfc"
+  depends_on "flock"
```
Comment (fxcoudert) [inline Formula/run-one.rb]: We don't have a `flock` formula in Homebrew core.

File: Formula/run-one.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+class RunOne < Formula
+  desc "Run one instance of a unique combination of command and args at a time"
+  homepage "https://launchpad.net/run-one"
+  url "https://github.com/datamade/run-one/archive/ea3d7744043bf737aa52c3093f068a49b7621d02.tar.gz"
```
Comment (fxcoudert) [inline Formula/run-one.rb]: > Latest version is 1.17

We only ship released versions.

File: Formula/run-one.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+class RunOne < Formula
+  desc "Run one instance of a unique combination of command and args at a time"
+  homepage "https://launchpad.net/run-one"
```
Comment (fxcoudert) [inline Formula/run-one.rb]: This does not look actively maintained…

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/run-one.rb (hunk 1 above)
Comment (fgregg) [reply Formula/run-one.rb]: Sorry I forgot that the dependency was only available through a cask. I'll withdraw this PR. https://github.com/discoteq/flock

**Comment by fgregg** [comment]
This depends upon flock, which is not part of homebrew-core. Closed this PR in favor of https://github.com/datamade/homebrew-datamade/blob/master/run-one.rb

**Pull request closed without merging**